backend/routes/user.routes.ts

```diff
@@ -0,0 +1,60 @@
+import express = require('express');
+import { User } from '../models/user.models';
+import { auth } from '../middleware/auth';
+
+export const user3Routes = express.Router();
+
+user3Routes.post('/register', async (req: any, res: any) => {
+  try {
+    const user = new User(req.body);
+    await user.save();
+    const token = await user.generateAuthToken();
+    res.status(201).send({user, token});
```

Comment:
ooops. I had been being lazy and just respond with '200' on success, hahaha

Reply:
It's important to send back at least the token for authentication. Angular will take this token, and assign it to local storage. This token state you are logged in and determines your identity, and even your device and browser identity at that.